src/cli.ts

```diff
@@ -173,6 +176,22 @@ function makeCommand (process: NodeJS.Process): Command {
     ).default(
       Enums.ComponentType.Application
     )
+  ).addOption(
```

Comment:
please move the worksapce-related options up so they appear right below the `omit`